arm-devtestlabs/2016-05-15/swagger/DTL.json

```diff
@@ -116,28 +175,95 @@
                     { "$ref": "#/parameters/subscriptionId" },
+                    { "$ref": "#/parameters/resourceGroupName" },
                     {
-                        "name": "resourceGroupName",
+                        "name": "labName",
                         "in": "path",
-                        "description": "The name of the resource group.",
+                        "description": "The name of the lab.",
                         "required": true,
                         "type": "string"
                     },
                     {
+                        "name": "$expand",
+                        "in": "query",
+                        "description": "Specify the $expand query. Example: 'properties($select=displayName)'",
+                        "required": false,
+                        "type": "string"
+                    },
+                    {
+                        "name": "$filter",
+                        "in": "query",
+                        "description": "The filter to apply to the operation.",
+                        "required": false,
+                        "type": "string"
+                    },
+                    {
+                        "name": "$top",
+                        "in": "query",
+                        "description": "The maximum number of resources to return from the operation.",
+                        "required": false,
+                        "type": "integer",
+                        "format": "int32"
+                    },
+                    {
+                        "name": "$orderBy",
+                        "in": "query",
+                        "description": "The ordering expression for the results, using OData notation.",
+                        "required": false,
+                        "type": "string"
+                    },
+                    { "$ref": "#/parameters/api-version" }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": { "$ref": "#/definitions/ResponseWithContinuation[ArtifactSource]" }
+                    },
+                    "default": {
+                        "description": "BadRequest",
+                        "schema": { "$ref": "#/definitions/CloudError" }
+                    }
+                },
+                "security": [ { "oauth2": [ "user_impersonation" ] } ],
+                "x-ms-pageable": { "nextLinkName": "nextLink" },
+                "x-ms-odata": "#/definitions/ArtifactSource"
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DevTestLab/labs/{labName}/artifactsources/{artifactSourceName}/armtemplates": {
+            "get": {
+                "tags": [ "ArmTemplate" ],
+                "description": "List azure resource manager templates in a given artifact source.",
+                "operationId": "ArmTemplate_List",

```

Comment:
Should be named as `ArmTemplate_ListByArtifactSource`

There are many of these linter violations, you may want to look into the naming. 

General Rule: 
>"The child tracked resource, 'X' with immediate parent 'Y', must have a list by immediate parent operation with operationId `_ListByY`